addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1006,7 +1006,7 @@ var UrlWidget = InputField.extend({
      * @private
      */
     _renderReadonly: function () {
-        this.$el.text(this.value)
+        this.$el.text(this.attrs.text || this.value)
```

Comment:
That's very strange ; why should a field widget take as value something else than its value?
Why using a field widget then, and not just a static link?